common/src/main/scala/com/gu/multimedia/storagetier/vidispine/SearchResultDocument.scala

```diff
@@ -58,6 +58,7 @@ object VSOnlineOutputMessage {
     val likelyFile = itemSimplified.item.shape.head.getLikelyFile
     val filePath = likelyFile.flatMap(_.getAbsolutePath)
     val fileSize = likelyFile.flatMap(_.sizeOption)
+    val projectIdAndContainingProjectIds = projectId +: itemSimplified.valuesForField("gnm_containing_projects", Some("Asset")).map(_.value).map(_.toInt)
```

Comment:
This is intentional to be guaranteed that we get the triggering `projectId` first in the list, so that we can filter it away later, because we don't need to fetch the status for that project.